### PR TITLE
RC 1.5.1

### PR DESCRIPTION
**Pull request by halge**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
 Changelog Application Jobsearch
 ===============================
-
 # 1.5.0
```
Comment (oskardrenske-ith) [inline CHANGELOG.md:3]: Add 1.5.1 and changes

**Review by lina1101 — APPROVED** [review]

**Review by oskardrenske-ith — CHANGES_REQUESTED** [review]
Update changelog.md with version 1.5.1 and info on changes

**Review by erik-gbg — COMMENTED** [review]
What Oskar said above, and that should be the only thing

**Review by erik-gbg — APPROVED** [review]

**Review by oskardrenske-ith — APPROVED** [review]

**Review by erik-gbg — APPROVED** [review]

**Review by lina1101 — APPROVED** [review]

**Review by halge — COMMENTED** [review]
ok

**Pull request merged**